k8s-daemonset/README.md

```diff
@@ -0,0 +1,35 @@
+# Hello World
+
+## linkerd-to-linkerd using Kubernetes daemonsets
+
+This is a sample application to demonstrate how to deploy a linkerd-to-linkerd
+configuration on Kubernetes using daemonsets.  The application consists of
+two python services: hello and world.  The hello service calls the world
+service.
+
+```
+hello -> linkerd (outgoing) -> linkerd (incoming) -> world
+
+```
+
+## Building
+
+Start by building the Docker image for the hello world application and pushing
+it to a repository that your Kubernetes cluster can access.
+
+```
+cd helloworld
+docker build -t <helloworld image name> .
+docker push <helloworld image name>
```

Comment:
TIOLI -- I think it would be fine to pre-build and push this image to a public repo on https://hub.docker.com/u/buoyantio/ -- maybe call it buoyantio/hello:latest or something like that.  Then you could hardcode that image in the k8s configs so they don't require manual editing.